spec/money_spec.rb

```diff
@@ -396,6 +388,14 @@
       expect(Money.new(1000, 'USD') <=> Money.new(2000, 'JPY')).to eq(-1)
       expect(Money.new(2000, 'JPY') <=> Money.new(1000, 'USD')).to eq(1)
     end
+
+    it "have the same hash value as $1" do
+      expect(money.hash).to eq(Money.new(1.00).hash)
+    end
+
+    it "does not have the same hash value as $2" do
+      expect(money.hash).to eq(Money.new(1.00).hash)
```

Comment:
test does not match the description

Reply:
This is just moving an existing test; figured that should be fixed in it's own PR?

Reply:
Although its small enough I suppose we can just fold it in